collection.php

```diff
@@ -0,0 +1,34 @@
+<?php
+//--QUERY--
+$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+$query = $db->prepare("SELECT `setName`, `setNumber`, `pieceCount`, `releaseYear` FROM `sets`;");
+$query->execute();
+$sets = $query->fetchAll();
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Lego Collection</title>
+</head>
+<body>
+<header>
+    <h1>My collection</h1>
+</header>
+
+<div class="main">
+    <div class="item">
+        <?php
+            foreach ($sets AS $set){
+                echo "<h2>" .$set['setName']. "</h2>";
+                echo "<h4>Set Number: " .$set['setNumber']. "</h4>";
+                echo "<h4>Piece Count: " .$set['pieceCount']. "</h4>";
+                echo "<h4>Release Year: " .$set['releaseYear']. "</h4>";
+            }
```

Comment:
And finally, I would pop this one into a function as well, one you can give the list of sets to, and it will return all of the HTML needed to display them. 

This function would be unit testable as well so would give you a chance to practice that in addition to creating nicer separation of concerns